src/sum.py

```diff
@@ -1,2 +1,11 @@
-def sum(a: int, b: int) -> int:
+def sum_int(a: int, b: int) -> int:
     return a + b
+
+def controllo(mess: list)->int:
+    n=""
+    while(n.isdecimal()==False): n=input(mess)
+    return int(n)
+n1=controllo("Inserire il primo numero: ")
+n2=controllo("Inserire il secondo numero: ")
+print("La somma tra "+str(n1)+"+"+str(n2)+"="+str(sum_int(n1,n2)))
```

Comment:
È buona pratica evitare di lasciare parti dello script senza indentazione,
perché verrebbero eseguiti ogni volta che qualcuno importa il tuo script.
Per evitarlo, aggiungi un 
```python
if __name__ == "__main__":
    main()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do).